_rules/non-visual-reference-alternative-825ule.md

```diff
@@ -0,0 +1,481 @@
+			<button onclick="alert('Surprise!')">Howdy</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 2
+
+The content in the second column is indicated with the word "right" (which is a location indicator based on visual perception) but also indicated by referencing that the content can be found below this content in the DOM order.
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	</style>
+</head>
+<body>
+
+<div class="col-container">
+  <div class="col">
+    <p>Interact with the content on the right (below this paragraph), for a surprise</p>
+  </div>
+
+  <div class="col">
+    <button onclick="alert('Surprise!')">Howdy</button>
+  </div>
+</div>
+
+</body>
+```
+
+#### Passed Example 3
+
+The content in the second column is a menu. The user is told to find the menu on the right (which is a visual indicator word) but the menu is also identified with the help of the "menu" heading above the content.
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the menu on the right, to navigate</p>
+		</div>
+		<div class="col">
+			<h1>Menu</h1>
+			<ul>
+				<li>
+					<a href="https://www.w3.org/Consortium/contact">Contact</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Help/">Help and FAQ</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Consortium/sponsor/">Donate</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Consortium/siteindex">Sitemap</a>
+				</li>
+			</ul>
+					
+		</div>
+	</div>
+</body>
+```
+
+
+#### Passed Example 4
+
+This document is using the word "square" but in this case it is no instruction.
+
+```html
+<body>
+	<p>In mathemathics a square is the result of multiplying a number by itself.</p>
+</body>
+```
+
+#### Passed Example 5
+
+The following text is tilted.
+
+```html
+<head>
+	<style> 
+		div.tilt {
+			height: 750px;
+			width: 150px;
+			-ms-transform: rotate(20deg); /* IE 9 */
+			-webkit-transform: rotate(20deg); /* Safari 3-8 */
+			transform: rotate(20deg);
+			}
+	</style>
+</head>
+<body>
+	<div class="tilt">This text is tilted.</div>
+</body>
+```
+
+#### Passed Example 6
+
+The button is indicated by the word "round". But the word is also included in the text of the element.
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click the round button, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Round button</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 7
+
+The image is indicated by the word narrow but the alt text of the image also includes the word narrow.
```

Comment:
Suggestion

The images are indicated by the words wide and narrow but the accessible names of the images also include the words wide and narrow.